services/validations/service.js

```diff
@@ -0,0 +1,33 @@
+const models = require('../../models');
```

Comment:
we should name it `utils`?

Reply:
and probably keep both `doesSuchSellerExist ` and `doesSuchServiceExist ` in one file?

Reply:
there might be more validations regarding seller and services in future so it kept it in separate files. Not sure if it's good idea. Your call.

Reply:
I think we should not complicate things for now. We can always separate them in future when there is enough complexity.

Reply:
okay, fair enough.